platform/client-admin/src/pages/ActionReportPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import moment, { unitOfTime } from "moment";
+import { Header } from '../components';
+import humanizeString from "humanize-string";
+import { RouteComponentProps } from 'react-router';
+import { useGetTodayActionReportQuery } from '../dataFacade';
+import { IonPage, IonContent, IonFooter, IonGrid, IonRow, IonCol, IonLoading } from '@ionic/react';
+
+
+const REPORT_COLUMN_SIZE = parseInt(process.env.REACT_APP_REPORT_COLUMN_SIZE || '4');
+const DAY_UNIT_OF_TIME: unitOfTime.Base = "day";
+
+export const ActionReportPage: React.FC<RouteComponentProps> = ({ match }) => {
+  const now = moment();
+  const todayMidnight = now.startOf(DAY_UNIT_OF_TIME);
+  const tomorrowMidnight = now.add(1,DAY_UNIT_OF_TIME).startOf(DAY_UNIT_OF_TIME);
+
+  const { data, loading, error } = useGetTodayActionReportQuery({
+    variables: {
+      todayMidnight,
+      tomorrowMidnight
+    }
+  });
+
+  if (error) {
+    console.log(error);
+  }
+
+  if (loading) return <IonLoading
+    isOpen={loading}
+    message={'Loading...'}
+  />;
+
+  const allActions = {
+    items: [...(data?.CreatedActions.items || []), ...(data?.AssignedActions.items || []), ...(data?.CompletedActions.items || [])]
+  };
+
+  const reports: any = {
+    TotalNumberActions: allActions,
+    ...data
+  };
+
+  const entries: any[] = Object.entries(reports)
+  const reportsRows = [];
+  for (let row = 0; row < entries.length; row += REPORT_COLUMN_SIZE) {
+    reportsRows.push(entries.slice(row, row + REPORT_COLUMN_SIZE));
+  }
+
+  const gridContent = reportsRows.map((reportRow, index) => {
+    const columns = reportRow.map((column) => {
+      return <IonCol key={column[0]}>
+        <div>
+          {humanizeString(column[0])}
+          <br/>
+          {column[1].items.length}
```

Comment:
Ok.. This is kinda badish and shows that aggregations are sometimes useful. We will have count implemented soon so hopefully that would do the job

Reply:
Yeap. Would be very useful. 

I had this crazy idea about implementing a `crud-stats-schema-codegen` plugin but decided against it.

Reply:
We have count available already in schema but it is not implemented